### DISCUSSION TITLE
PyPy new version introduced 29.03.2022 causes failure

### DISCUSSION THREAD

**Discussion by freyrfinnboga** [actions/virtual-environments#5316] [General questions] [answered]
Has anyone noticed issue with the new PyPy release introduced yesterday https://downloads.python.org/pypy/
So the default listed here https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md as 3.7.12 now bumps up to 3.7.13 and the Toolset.Tests.ps1 fails on line 39 with
Context 3.7
vhd:    [+] tool-cache directory exists 8ms (5ms|3ms)
vhd:    [+] python.exe --version 71ms (70ms|2ms)
vhd:    [-] Scripts\pip.exe --version 51ms (47ms|4ms)
vhd:     Command 'C:\hostedtoolcache\windows\PyPy\3.7.13\x86\Scripts\pip.exe --version' has finished with exit code
vhd:         'C:\hostedtoolcache\windows\PyPy\3.7.13\x86\Scripts\pip.exe' is not recognized as an internal or external command,
vhd:         operable program or batch file.
vhd:     at "$binaryFullPath $Arguments" | Should -ReturnZeroExitCode, C:\image\Tests\Toolset.Tests.ps1:39
vhd:     at <ScriptBlock>, C:\image\Tests\Toolset.Tests.ps1:39

**Comment by miketimofeev** [comment, marked as answer]
Hi @freyrfinnboga!
This is already fixed in https://github.com/actions/virtual-environments/pull/5308

**Reply by freyrfinnboga** [reply]
That's fantastic thank you for the quick reply
